deployment/helm/synopsys-alert/README.md

```diff
@@ -130,3 +190,237 @@ Alternatively, a YAML file that specifies the values for the above parameters ca
 ```console
 $ helm install . --name <name> --namespace <namespace> --set enableStandalone=true
 ```
+
+### Persistent Storage
+The section will describe the changes needed to configure persistent storage.  
+**Please note:** By default the 'values.yaml' file has persistent storage disabled in order for Alert to work out of the box.
+
+#### Enable Persistent Storage
+- In the 'values.yaml' file change 
+  ```yaml
+  enablePersistentStorage: true 
+  ```
+#### With Storage Claims
+This section defines configuration using Persistent Volume Claims.  Claims can be optionally used rather than just a Persistent volume.
+You must have a claim created for the 'alert' service regardless of an on-premise or external database.
+For the on-premise database deployment a second Persistent Volume Claim must be created to store the database data.
+
+##### Alert Volume Claim Configuration
+- A persistent volume must be created first
```

Comment:
It only needs to be created first if they don't have a dynamic provisioner - which will see the pvc and create a persistent volume for it